timescaledb/getting-started/add-data.md

```diff
@@ -1,53 +1,63 @@
-# 4. Add time-series data
+# Add time-series data
 
-To showcase TimescaleDB and get you familiar with its features, we’ll need some
-sample data to play around with. We’ll use the real-world scenario of climate
-change. Data about the climate in a certain region is time-series data, as it
-describes how the climate in that area changes over time.
+To help get you familiar with TimescaleDB's features, you’ll need some
+sample data to use. For this tutorial, we provide real-time 
+stock trade data (aka. "tick data") provided by [twelvedata][twelve-data] for 100 stock symbols for the most recent month. 
 
-## About the dataset
+### About the dataset
 
-Our dataset comes from [OpenWeatherMap](https://openweathermap.org) and contains
-measurements for 10 cities in which Timescalers reside: New York City, San
-Francisco, Princeton, Austin, Stockholm, Lisbon, Pietermaritzburg, Nairobi,
-Toronto, and Vienna.
+The dataset contains tick level (second by second) stock trade data for the top 100 most traded symbols along with a separate table of company symbols and company names. 
+There are two tables that you will ingest, a hypertable `stocks_real_time` and a regular table `company`. 
 
-The dataset contains weather metrics for each city from 1 January 1979 to 27 April 2021.
+Details on the data set are as follows:
+`stocks_real_time`: Stock data indicating quotes for stock prices for a given second during trading hours
 
-For each city, we record the following metrics:
 ```bash
```

Comment:
Does it make sense to mention datatypes of these columns?

Reply:
I think that is a great idea!

Reply:
I think that is a great idea!